db/migrate/20160114200145_add_mirror_on_sync_to_repositories.rb

```diff
@@ -0,0 +1,6 @@
+class AddMirrorOnSyncToRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :mirror_on_sync, :boolean, :default => true, :null => false
+    update "UPDATE katello_repositories SET mirror_on_sync = 'f'"
```

Comment:
That I get, I am just less clear on the line of code, I am used to us doing updates like that through a class that represents the model here in the migration vs exact SQL code.